zathura/commands.c

```diff
@@ -149,6 +149,69 @@ bool cmd_bookmark_open(girara_session_t* session, girara_list_t* argument_list)
   return true;
 }
 
+bool cmd_jumplist_list(girara_session_t* session, girara_list_t* argument_list) {
+  g_return_val_if_fail(session != NULL, false);
+  g_return_val_if_fail(session->global.data != NULL, false);
+  zathura_t* zathura = session->global.data;
+  if (zathura->document == NULL) {
+    girara_notify(session, GIRARA_ERROR, _("No document opened."));
+    return false;
+  }
+
+  int num_entries = 5;
+
+  const unsigned int argc = girara_list_size(argument_list);
+  if (argc == 1) {
+    const char* value = girara_list_nth(argument_list, 0);
+    if (value != NULL) {
+      if (atoi(value) == 0) {
```

Comment:
The result of this operation can be stored and does not need to be repeated.

Also we mostly use `strto*` to convert strings to integers as it has better error handling.